### ISSUE TITLE
[a11y]: Tooltip content not being read by screen reader

### ISSUE THREAD

**Issue by arimori** [carbon-design-system/carbon#11873]
### Package

carbon-components-react

### Browser

Chrome

### Operating System

Windows

### Package version

7.55

### React version

16.13.1

### Automated testing tool and ruleset

Google Chrome Screen Reader

### Assistive technology

Google Chrome Screen Reader

### Description

Tooltip content not being read by Chrome Screen-reader.

### Reproduction/example

https://v10-react.carbondesignsystem.com/?path=/story/components-tooltip--default-bottom

### Steps to reproduce

- Click in the link above;
- Start Chrome Screen-reader;
- Click on the tooltip icon.

Expected: Chrome screen-reader to be able to read the tooltip content.
Result: Chrome screen-reader does not read the tooltip content.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by tay1orjones** [comment]
@arimori what screen reader in chrome are you using? Is it [this extension](https://chrome.google.com/webstore/detail/screen-reader/kgejglhpjiefppelpmljglcjbhoiplfn?hl=en), maybe?


We primarily focus on JAWS, NVDA, and Voiceover. Are you seeing the same issue in those environments?

**Comment by arimori** [comment]
Exactly, I'm using the screen reader from google for now, but I already raised an issue with my team to get a valid license to use JAWS so that I can test in this different environment.
Basically the screen reader doesn't read the text when the tooltip gains focus. It only reads when the tooltip content is clicked.
Also I can't get to this text with keyboard (neither arrows nor tab)

**Comment by arimori** [comment]
The QA internal team of my project retested and closed this issue since they were able to use JAWS to read the tooltip content.
So I think we can close this issue now.

Thanks @tay1orjones!
